cog/daily_charge.py

```
@@ -23,95 +26,242 @@
 
 
 class Charge(commands.Cog):
-    def __init__(self, bot):
+    def __init__(self, bot: discord.Bot):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)